src/lib/constants.ts

```diff
@@ -2,6 +2,14 @@ export const PAGE_LIMIT = 12; // default page limit
 export const CARD_LIMIT = 6; // default card limit
 export const INTERVAL = 5 * 60000; // default interval to check for feedback
 
+export const REGION_FRA = 'fra';
+export const REGION_SYD = 'syd';
+export const REGION_NYC = 'nyc';
+
+export const SUBDOMAIN_FRA = 'fra.';
+export const SUBDOMAIN_SYD = 'syd.';
+export const SUBDOMAIN_NYC = 'nyc.';
```

Comment:
I dont think we need constants for that, its good enough to have them locally in the `sdk.ts`